db/app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const connection = require('../db/connection');
+const { getCategories } = require('../db/controllers/categories.controller');
+
+app.get('/api/categories', getCategories);
+
+app.all("*", (req, res) => {
+    res.status(404).send({ msg: "Invalid endpoint" });
+})
+
+app.use((err, req, res, next) => {
```

Comment:
Technically don't need this error handling middleware at the moment, it's not serving a purpose. we should add detail and features as and when we need them as good practice.

Reply:
No need to remove them though